vscode-extensions.snakemake.snakemake-lang: init at 0.1.8

jrhawley opened the pull request:
## Description of changes

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [X] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

drupol committed:
asciinema-automation: init at 0.2.0

trzpiot committed:
vscode-extensions.ms-azuretools.vscode-docker: 1.24.0 -> 1.29.0

- https://github.com/microsoft/vscode-docker/releases/tag/v1.29.0
- https://github.com/microsoft/vscode-docker/compare/v1.24.0...v1.29.0

jrhawley committed:
vscode-extensions.snakemake.snakemake-lang: init at v0.1.8

blinry committed:
tic-80: init at 1.1.2837

r-ryantm committed:
hello-wayland: unstable-2023-10-26 -> 0-unstable-2024-03-04

bryanasdev000 committed:
irpf: 2023-1.5 -> 2024-1.0

Qusic committed:
st: 0.9 -> 0.9.1

ocfox committed:
pwvucontrol: 0.2 -> 0.3.1

Guanran928 committed:
yesplaymusic: 0.4.7 -> 0.4.8-2

Guanran928 committed:
yesplaymusic: move to by-name

superherointj commented on pkgs/applications/editors/vscode/extensions/default.nix:
```diff
@@ -3368,6 +3368,22 @@ let
+          downloadPage = "https://marketplace.visualstudio.com/items?itemName=snakemake.snakemake-lang";
+          homepage = "https://github.com/snakemake/snakemake-lang-vscode-plugin";
+          license = lib.licenses.mit;
+        };
```
```suggestion
         maintainers = with lib.maintainers; [ ];
        };
```
Consider adding yourself as a package maintainer. (Optional)

superherointj commented on pkgs/applications/editors/vscode/extensions/default.nix:
```diff
@@ -3368,6 +3368,22 @@ let
+        };
+        meta =  {
+          changelog = "https://github.com/snakemake/snakemake-lang-vscode-plugin/blob/master/CHANGELOG.md";
+          description = "Language support and snippets for Snakemake workflows (Snakefile, *.smk).";
```
```suggestion
          description = "A Visual Studio Code extension that adds language support and snippets for Snakemake workflows";
```

superherointj commented:
Please rebase.

Ramblurr committed:
davis: init at 4.4.1

Release: https://github.com/tchapi/davis/releases/tag/v4.4.1

Ramblurr committed:
nixos/davis: init

eliandoran committed:
ut1999: add support for i686-linux

kurnevsky committed:
ubpm: 1.7.3 -> 1.10.0

alyssais committed:
linuxPackages.rust-out-of-tree-module: tighten platforms

Linux only supports Rust on x86_64 and (as of 6.9) aarch64.

alyssais committed:
nixosTests.kernel-rust.linux_testing: init for aarch64

Older kernels do not support Rust on aarch64, so we have to move the
platform check from all-tests.nix into the test itself.

katanallama committed:
maintainers: add katanallama

katanallama committed:
vscode-extensions.ms-toolsai.datawrangler: init at 0.29.6

eliandoran committed:
innoextract: Enable on darwin

eliandoran committed:
ut1999: Enable darwin support

drupol committed:
rabbit: init at 1.0.0

eliandoran committed:
ut1999: add aarch64_linux support

Commit:
cudaPackages.saxpy: use pkgs.autoAddDriverRunpath

Commit:
llama-cpp: use pkgs.autoAddDriverRunpath

Commit:
python3Packages.torch*: use pkgs.autoAddDriverRunpath

GaetanLepage committed:
vimPlugins.qmk-nvim: init at 2024-02-15

jcumming committed:
bluespec: 2023.01 -> 2024.01

Aleksanaa committed:
go-musicfox: 4.3.1 -> 4.3.3

soispha committed:
nixos/libreddit: Don't hard code binary name in `ExecStart`

Hard coding the name to `libreddit` makes it impossible to use
this module to run `redlib` (a continued fork of `libreddit`).
As `libreddit` has the `mainProgram` meta attribute set, changing
this to `lib.getExe` should not change the module for `libreddit`,
but allows it to work with the `redlib` package.

adminy committed:
rtl-sdr: 0.8.0 -> 0.9.0

Bump rtl-sdr to newer version to support RTL-SDR.com V4 dongle & get
improvements.

happysalada committed:
Merge pull request #297014 from onny/davfs2

nixos/davfs2: Convert extraConfig to freeform type (RFC42)

NickCao committed:
Merge pull request #300344 from wegank/sarasa-gothic-bump

sarasa-gothic: 1.0.8 -> 1.0.9

NickCao committed:
Merge pull request #300337 from r-ryantm/auto-update/bacon

bacon: 2.15.0 -> 2.16.0

NickCao committed:
Merge pull request #300323 from xrelkd/update/cargo-expand

cargo-expand: 1.0.80 -> 1.0.81

mweinelt committed:
Merge pull request #300415 from ilyagr/pipx-completion

`pipx`: add shell completions

mweinelt committed:
Merge pull request #290779 from savyajha/radian-update

python311Packages.radian: 0.6.8 -> 0.6.12

NickCao committed:
Merge pull request #300316 from r-ryantm/auto-update/kraft

kraft: 0.7.14 -> 0.8.1

NickCao committed:
Merge pull request #300315 from r-ryantm/auto-update/python311Packages.urwid

python311Packages.urwid: 2.6.8 -> 2.6.10

mweinelt committed:
Merge pull request #296444 from wolfgangwalther/sphinx-rtd-dark-mode

python3Packages.sphinx-rtd-dark-mode: init at 1.3.0

NickCao committed:
Merge pull request #300355 from r-ryantm/auto-update/python312Packages.lion-pytorch

python312Packages.lion-pytorch: 0.1.2 -> 0.1.4

NickCao committed:
Merge pull request #300352 from r-ryantm/auto-update/git-cliff

git-cliff: 2.1.2 -> 2.2.0

NickCao committed:
Merge pull request #300422 from r-ryantm/auto-update/python311Packages.awscrt

python311Packages.awscrt: 0.20.5 -> 0.20.6

NickCao committed:
Merge pull request #300419 from r-ryantm/auto-update/python311Packages.dtw-python

python311Packages.dtw-python: 1.3.1 -> 1.4.2

NickCao committed:
Merge pull request #300408 from r-ryantm/auto-update/saml2aws

saml2aws: 2.36.14 -> 2.36.15

NickCao committed:
Merge pull request #300405 from r-ryantm/auto-update/python311Packages.qbittorrent-api

python311Packages.qbittorrent-api: 2024.2.59 -> 2024.3.60

r-ryantm committed:
oxker: 0.6.1 -> 0.6.2

NickCao committed:
Merge pull request #300397 from r-ryantm/auto-update/python312Packages.tplink-omada-client

python312Packages.tplink-omada-client: 1.3.12 -> 1.3.13

NickCao committed:
Merge pull request #300396 from r-ryantm/auto-update/obs-studio-plugins.obs-move-transition

obs-studio-plugins.obs-move-transition: 2.11.0 -> 2.12.0

NickCao committed:
Merge pull request #300486 from r-ryantm/auto-update/funzzy

funzzy: 1.1.1 -> 1.2.0

NickCao committed:
Merge pull request #300469 from r-ryantm/auto-update/dualsensectl

dualsensectl: 0.4 -> 0.5

imincik committed:
Merge pull request #300347 from marsam/update-pgrouting

postgresqlPackages.pgrouting: 3.6.1 -> 3.6.2

NickCao committed:
Merge pull request #300452 from fabaff/tesla-fleet-api-bump

python312Packages.tesla-fleet-api: 0.5.3 -> 0.5.6

NickCao committed:
Merge pull request #300451 from fabaff/qovery-cli-bump

qovery-cli: 0.86.0 -> 0.86.2

NickCao committed:
Merge pull request #300450 from fabaff/pyunifiprotect-bump

python311Packages.pyunifiprotect: 5.1.1 -> 5.1.2

NickCao committed:
Merge pull request #300449 from fabaff/tplink-omada-client-bump

 python312Packages.tplink-omada-client: 1.3.12 -> 1.3.13

NickCao committed:
Merge pull request #300488 from r-ryantm/auto-update/oxker

oxker: 0.6.1 -> 0.6.2

NickCao committed:
Merge pull request #300466 from r-ryantm/auto-update/python311Packages.allure-python-commons-test

python311Packages.allure-python-commons-test: 2.13.3 -> 2.13.4

NickCao committed:
Merge pull request #300368 from r-ryantm/auto-update/freetds

freetds: 1.4.10 -> 1.4.11

NickCao committed:
Merge pull request #300276 from r-ryantm/auto-update/treesheets

treesheets: unstable-2024-03-14 -> unstable-2024-03-30

NickCao committed:
Merge pull request #300250 from r-ryantm/auto-update/renode-dts2repl

renode-dts2repl: unstable-2024-03-22 -> unstable-2024-03-24

NickCao committed:
Merge pull request #300243 from r-ryantm/auto-update/commitizen

commitizen: 3.20.0 -> 3.21.3

NickCao committed:
Merge pull request #300236 from r-ryantm/auto-update/xray

xray: 1.8.7 -> 1.8.10

NickCao committed:
Merge pull request #300219 from r-ryantm/auto-update/litecli

litecli: 1.10.0 -> 1.10.1

NickCao committed:
Merge pull request #300173 from r-ryantm/auto-update/python311Packages.qpsolvers

python311Packages.qpsolvers: 4.3.1 -> 4.3.2

NickCao committed:
Merge pull request #300148 from r-ryantm/auto-update/kuma-experimental

kuma-experimental: 2.6.2 -> 2.6.3

NickCao committed:
Merge pull request #300147 from r-ryantm/auto-update/kuma-cp

kuma-cp: 2.6.2 -> 2.6.3

NickCao committed:
Merge pull request #300131 from r-ryantm/auto-update/comrak

comrak: 0.21.0 -> 0.22.0

NickCao committed:
Merge pull request #299891 from r-ryantm/auto-update/python312Packages.mitogen

python312Packages.mitogen: 0.3.4 -> 0.3.5

nixpkgs-merge-bot[bot] committed:
Merge pull request #294036 from r-ryantm/auto-update/zabbix-agent2-plugin-postgresql

zabbix-agent2-plugin-postgresql: 6.0.25 -> 6.4.12

r-ryantm committed:
svd2rust: 0.32.0 -> 0.33.0

r-ryantm committed:
trippy: 0.9.0 -> 0.10.0

NickCao committed:
Merge pull request #299653 from illustris/cloud-init

cloud-init: update patch

r-ryantm committed:
dovecot_fts_xapian: 1.7.9 -> 1.7.10

r-ryantm committed:
bun: 1.0.35 -> 1.0.36

fabaff committed:
Merge pull request #300447 from fabaff/speechrecognition-bump

python312Packages.speechrecognition: 3.10.2 -> 3.10.3

r-ryantm committed:
python311Packages.astropy: 6.0.0 -> 6.0.1

emilylange committed:
Merge pull request #300342 from stuebinm/mattermost-9.5.3

mattermost: 9.5.2 → 9.5.3

doronbehar committed:
Merge pull request #296835 from Guanran928/mpv-oscs

mpv-osc-modern, modernx, modernx-zydezu: init

DCsunset committed:
pandoc-include: 1.3.0 -> 1.3.1

nixpkgs-merge-bot[bot] committed:
Merge pull request #300433 from r-ryantm/auto-update/home-manager

home-manager: unstable-2024-03-22 -> unstable-2024-03-31

Lassulus committed:
Merge pull request #300351 from r-ryantm/auto-update/fvwm3

fvwm3: 1.0.9 -> 1.1.0

wegank committed:
Merge pull request #300142 from SomeoneSerge/fix/rerun/darwin-iconv

python3Packages.rerun-sdk: fix -liconv on darwin

wegank committed:
Merge pull request #298338 from jakuzure/glasskube-fix-bundle

glasskube: fix missing bundles from npm

dotlambda committed:
warp: broken on Darwin

dotlambda committed:
Merge pull request #300508 from dotlambda/warp-darwin

warp: broken on Darwin

7c6f434c committed:
Merge pull request #300065 from hraban/sbcl-2.4.3

sbcl: 2.4.2 -> 2.4.3

amaxine committed:
Merge pull request #299832 from HannahMR/aperture0.3

drupol committed:
typstyle: 0.11.5 -> 0.11.8

vbgl committed:
doc: clean-up Coq derivation example (#299927)

DanielSidhion committed:
Merge pull request #300019 from eflanagan0/doc-lowercase-npm

doc: lowercase npm name

r-ryantm committed:
obs-studio: 30.1.0 -> 30.1.1

r-ryantm committed:
mame: 0.263 -> 0.264

drupol committed:
Merge pull request #300453 from r-ryantm/auto-update/iosevka-bin

iosevka-bin: 29.0.2 -> 29.0.5

r-ryantm committed:
python311Packages.llama-index-agent-openai: 0.2.0 -> 0.2.1

wegank committed:
Merge pull request #297735 from Guanran928/yesplaymusic

yesplaymusic: 0.4.7 -> 0.4.8-2

fabaff committed:
checkov: 3.2.49 -> 3.2.50

Diff: https://github.com/bridgecrewio/checkov/compare/refs/tags/3.2.49...3.2.50

Changelog: https://github.com/bridgecrewio/checkov/releases/tag/3.2.50

r-ryantm committed:
python311Packages.elementpath: 4.3.0 -> 4.4.0

SuperSandro2000 committed:
Merge pull request #300223 from r-ryantm/auto-update/gitstatus

gitstatus: 1.5.4 -> 1.5.5

stigtsp committed:
Merge pull request #300461 from stigtsp/perl/http-body-1.23

perlPackages.HTTPBody: 1.22 -> 1.23

r-ryantm committed:
raft-cowsql: 0.22.0 -> 0.22.1

SuperSandro2000 committed:
Merge pull request #299724 from wineee/wlroots_0_15

{hikari, wlroots_0_15}: remove

SuperSandro2000 committed:
Merge pull request #298801 from Qusic/slstatus

slstatus: add updateScript

SuperSandro2000 committed:
Merge pull request #295646 from jnsgruk/rockcraft

rockcraft: init at 1.2.2

AndersonTorres committed:
fvwm2: migrate to by-name

AndersonTorres committed:
fvwm3: migrate to by-name

SuperSandro2000 committed:
Merge pull request #298681 from SuperSandro2000/pyzbar-dev

python311Packages.pyzbar: don't propagate library with dev dependencies

AndersonTorres committed:
fvwm3: refactor

- use lib.enableFeature
- strictDeps
- get rid of nested with
- meta.longDescription
- meta.changelog

fabaff committed:
Merge pull request #300522 from r-ryantm/auto-update/python311Packages.llama-index-agent-openai

python311Packages.llama-index-agent-openai: 0.2.0 -> 0.2.1

NickCao committed:
Merge pull request #300454 from r-ryantm/auto-update/crystalline

crystalline: 0.12.1 -> 0.12.2

r-ryantm committed:
bash-supergenpass: unstable-2020-02-03 -> unstable-2024-03-24

drupol committed:
Merge pull request #300209 from drupol/rabbit/init/1-0-0

RABBIT: init at 1.0.0

Atemu committed:
Merge pull request #300203 from eliandoran/dev/ut1999_multiplatform

ut1999: add x86_64-darwin, aarch64-linux, i686-linux support, innoextract: enable on darwin

veprbl committed:
Merge pull request #294826 from xworld21/texlive-install-tl

texlive.withPackages: replace postBuild with install-tl like script

dotlambda committed:
python312Packages.python-songpal: 0.16.1 -> 0.16.2

Diff: https://github.com/rytilahti/python-songpal/compare/refs/tags/release/0.16.1...0.16.2

Changelog: https://github.com/rytilahti/python-songpal/blob/release/0.16.2/CHANGELOG.md

wegank committed:
Merge pull request #300045 from Aleksanaa/ascii-draw

ascii-draw: init at 0.3.0

dotlambda committed:
Merge pull request #299197 from SuperSandro2000/vaultwarden-webvault

vaultwarden.webvault: 2024.1.2b -> 2024.3.0

drupol committed:
Merge pull request #300518 from r-ryantm/auto-update/obs-studio

obs-studio: 30.1.0 -> 30.1.1

drupol committed:
Merge pull request #300516 from r-ryantm/auto-update/mame

mame: 0.263 -> 0.264

bachp committed:
Merge pull request #300350 from r-ryantm/auto-update/minio

minio: 2024-03-15T01-07-19Z -> 2024-03-26T22-10-45Z

drupol committed:
Merge pull request #300525 from fabaff/checkov-bump

checkov: 3.2.49 -> 3.2.50

wegank committed:
Merge pull request #300493 from r-ryantm/auto-update/svd2rust

svd2rust: 0.32.0 -> 0.33.0

wegank committed:
Merge pull request #300463 from soispha/libreddit_fix_hardcoded_binary_name

nixos/libreddit: Don't hard code binary name in `ExecStart`

wegank committed:
Merge pull request #299968 from wegank/gurobi-bump

gurobi, python312Packages.gurobipy: 9.5.1 -> 11.0.1

wegank committed:
Merge pull request #300267 from khaneliman/yabai

yabai: 7.0.3 -> 7.0.4

wegank committed:
Merge pull request #299087 from wegank/aria-alias

aria2: deprecate aria alias

wegank committed:
Merge pull request #300494 from r-ryantm/auto-update/trippy

trippy: 0.9.0 -> 0.10.0

colemickens committed:
Revert "trezor-suite: mark as broken"

This reverts commit 2beb7de23e38e81a0b2948a32243a1b958ca81dc.

wegank committed:
Merge pull request #299497 from r-ryantm/auto-update/mihomo

mihomo: 1.18.1 -> 1.18.2

wegank committed:
Merge pull request #300533 from atorres1985-contrib/fvwm

Fvwm: refactor and migrate to by-name

wegank committed:
Merge pull request #298149 from Vinetos/add-quarkus

quarkus: init at 3.8.3

DerDennisOP committed:
bitwarden-directory-connector: 2024.3.1 -> 2024.3.2

wegank committed:
Merge pull request #300561 from colemickens/colemickens/trezor-unbroken

Revert "trezor-suite: mark as broken"

wegank committed:
Merge pull request #299103 from r-ryantm/auto-update/sitespeed-io

sitespeed-io: 33.3.0 -> 33.4.0

wegank committed:
Merge pull request #299073 from r-ryantm/auto-update/gfxreconstruct

gfxreconstruct: 1.0.2 -> 1.0.3

wegank committed:
Merge pull request #298927 from r-ryantm/auto-update/gnss-share

gnss-share: 0.8.0 -> 0.8.1

wegank committed:
Merge pull request #298953 from r-ryantm/auto-update/bambu-studio

bambu-studio: 01.08.04.51 -> 01.09.00.60

wegank committed:
Merge pull request #299059 from r-ryantm/auto-update/coreth

coreth: 0.13.1 -> 0.13.2

wegank committed:
Merge pull request #297696 from r-ryantm/auto-update/runelite

runelite: 2.6.13 -> 2.7.1

wegank committed:
Merge pull request #297533 from r-ryantm/auto-update/neo4j

neo4j: 5.18.0 -> 5.18.1

r-ryantm committed:
werf: 1.2.300 -> 1.2.301

wegank committed:
Merge pull request #297577 from r-ryantm/auto-update/mongodb-compass

mongodb-compass: 1.42.2 -> 1.42.3

wegank committed:
Merge pull request #297579 from r-ryantm/auto-update/libucl

libucl: 0.9.0 -> 0.9.1

wegank committed:
Merge pull request #297522 from r-ryantm/auto-update/optifine

optifine: 1.20.1_HD_U_I6 -> 1.20.4_HD_U_I7

wegank committed:
Merge pull request #297314 from r-ryantm/auto-update/dwm

dwm: 6.4 -> 6.5

wegank committed:
Merge pull request #294655 from chuangzhu/pure-protobuf-3.0.1

python3Packages.pure-protobuf: 2.3.0 -> 3.0.1

wegank committed:
Merge pull request #292969 from r-ryantm/auto-update/komikku

komikku: 1.38.1 -> 1.39.0

adamcstephens committed:
Merge pull request #298981 from mahmoudk1000/master

anki-bin: clear plugin path environment variable

wegank committed:
Merge pull request #297595 from r-ryantm/auto-update/uhdm

uhdm: 1.80 -> 1.82

wegank committed:
Merge pull request #276432 from r-ryantm/auto-update/surelog

surelog: 1.80 -> 1.82

dotlambda committed:
Merge pull request #300491 from r-ryantm/auto-update/python312Packages.python-songpal

python312Packages.python-songpal: 0.16.1 -> 0.16.2

dotlambda committed:
python311Packages.herepy: 3.6.0 -> 3.6.1

Diff: https://github.com/abdullahselek/HerePy/compare/refs/tags/3.6.0...3.6.1

Changelog: https://github.com/abdullahselek/HerePy/releases/tag/3.6.1

pinpox committed:
wastebin: init at 2.4.3 (#287455)

* wastebin: init at 2.4.3

* nixos/wastebin: init

Add module and test for wastebin

r-ryantm committed:
websocat: 1.12.0 -> 1.13.0

dotlambda committed:
Merge pull request #300585 from r-ryantm/auto-update/python311Packages.herepy

python311Packages.herepy: 3.6.0 -> 3.6.1

r-ryantm committed:
johnny-reborn-engine: 0.30 -> 0.34

Qusic committed:
st: add qusic to maintainers

r-ryantm committed:
rabbit: 1.0.0 -> 1.0.1

adamcstephens committed:
Merge pull request #300151 from adamcstephens/lxcfs/6

lxcfs: 5.0.4 -> 6.0.0

r-ryantm committed:
quarkus: 3.8.3 -> 3.9.1

r-ryantm committed:
python312Packages.plexapi: 4.15.10 -> 4.15.11

r-ryantm committed:
python312Packages.imbalanced-learn: 0.12.0 -> 0.12.2

r-ryantm committed:
conftest: 0.50.0 -> 0.51.0

r-ryantm committed:
python312Packages.pytest-mypy-plugins: 3.1.1 -> 3.1.2

r-ryantm committed:
zbus-xmlgen: 4.0.1 -> 4.1.0

r-ryantm committed:
vulkan-caps-viewer: 3.34 -> 3.40

r-ryantm committed:
fishPlugins.forgit: 24.03.2 -> 24.04.0

r-ryantm committed:
chezmoi: 2.47.2 -> 2.47.3

r-ryantm committed:
python311Packages.instructor: 0.6.4 -> 0.6.8

r-ryantm committed:
kubedb-cli: 0.42.1 -> 0.44.0

drupol committed:
Merge pull request #300599 from r-ryantm/auto-update/rabbit

rabbit: 1.0.0 -> 1.0.1

r-ryantm committed:
sickgear: 3.30.15 -> 3.30.16

Mic92 committed:
Merge pull request #300622 from r-ryantm/auto-update/python311Packages.instructor

python311Packages.instructor: 0.6.4 -> 0.6.8

r-ryantm committed:
oxlint: 0.2.14 -> 0.2.15

ulrikstrid committed:
Merge pull request #300220 from mschwaig/fix-rocm-deprecation-notices

rocmPackages: fix deprecation notices

r-ryantm committed:
tailwindcss: 3.4.1 -> 3.4.3

kirillrdy committed:
Merge pull request #299440 from kurnevsky/ubpm110

ubpm: 1.7.3 -> 1.10.0

SebTM committed:
sysdig: 0.35.3 -> 0.36.0

SebTM committed:
sysdig: fix formattig

r-ryantm committed:
kopia: 0.16.0 -> 0.16.1

SuperSandro2000 committed:
Merge pull request #297472 from Qusic/master

st: 0.9 -> 0.9.1

r-ryantm committed:
templ: 0.2.646 -> 0.2.648

GaetanLepage committed:
invidious: 0.20.1-unstable-2024-03-08 -> 0.20.1-unstable-2024-03-31

fabaff committed:
python312Packages.pytest-mypy-plugins: refactor

fabaff committed:
python312Packages.pytest-mypy-plugins: update disabled

fabaff committed:
python312Packages.plexapi: refactor

drupol committed:
Merge pull request #265695 from drupol/asciinema_automation/init/0-1-2

asciinema-automation: init at 0.2.0

doronbehar committed:
Merge pull request #300514 from r-ryantm/auto-update/python311Packages.astropy

python311Packages.astropy: 6.0.0 -> 6.0.1

drupol committed:
Merge pull request #300511 from drupol/typstyle/bump/0-11-8

typstyle: 0.11.5 -> 0.11.8

ck3d committed:
Merge pull request #299798 from ck3d/localai-2110

local-ai: 2.10.1 -> 2.11.0

drupol committed:
github-desktop: add missing phase hooks

Mic92 committed:
sysdig: disable hardened and zen kernels

Also it currently compiles with zen kernels, it doesn't with lqx.
Since we currently don't have a mechanism of lqx only, we can re-add,
if one of the lqx/zen maintainers steps up and also help maintaining and testing sysdig.

Mic92 committed:
Merge pull request #300630 from SebTM/sysdig_036

sysdig: 0.35.3 -> 0.36.0

happysalada committed:
Merge pull request #300615 from r-ryantm/auto-update/fishPlugins.forgit

fishPlugins.forgit: 24.03.2 -> 24.04.0

drupol committed:
github-desktop: use `stdenvNoCC`

drupol committed:
github-desktop: sort `meta` attributes

drupol committed:
github-desktop: avoid reusing `pname` everywhere

drupol committed:
github-desktop: move to `pkgs/by-name`

SuperSandro2000 committed:
Merge pull request #300217 from DerDennisOP/cgoban

cgoban: 3.5.23 -> 3.5.144

Raroh73 committed:
vscode-extensions.continue.continue: 0.8.12 -> 0.8.22

drupol committed:
github-desktop: 3.3.10 -> 3.3.12

Now support `aarch64` and `x86_64` archs

peterhoeg committed:
kwin & plasma-mobile: 6.0.3 -> 6.0.3.1

adamcstephens committed:
Merge pull request #300152 from adamcstephens/cowsql/1.15.6

cowsql: 0.15.4 -> 0.15.6

Atemu committed:
Merge pull request #291129 from blinry/tic-80

tic-80: init at 1.1.2837

GaetanLepage committed:
python311Packages.testcontainers: 4.2.0 -> 4.3.0

Changelog:
https://github.com/testcontainers/testcontainers-python/releases/tag/testcontainers-v4.3.0

adamcstephens committed:
Merge pull request #300633 from r-ryantm/auto-update/tailwindcss

tailwindcss: 3.4.1 -> 3.4.3

Commit:
Merge pull request #300643 from GaetanLepage/invidious

invidious: 0.20.1-unstable-2024-03-08 -> 0.20.1-unstable-2024-03-31

K900 committed:
Merge pull request #300667 from peterhoeg/u/kwin_mobile

kwin & plasma-mobile: 6.0.3 -> 6.0.3.1

leona-ya committed:
Merge pull request #300642 from r-ryantm/auto-update/templ

templ: 0.2.646 -> 0.2.648

r-ryantm committed:
act: 0.2.60 -> 0.2.61

adamcstephens committed:
Merge pull request #300530 from r-ryantm/auto-update/raft-cowsql

raft-cowsql: 0.22.0 -> 0.22.1

NickCao committed:
Merge pull request #300528 from r-ryantm/auto-update/python311Packages.elementpath

python311Packages.elementpath: 4.3.0 -> 4.4.0

NickCao committed:
Merge pull request #300504 from DCsunset/update-pandoc-include

pandoc-include: 1.3.0 -> 1.3.1

NickCao committed:
Merge pull request #300499 from r-ryantm/auto-update/bun

bun: 1.0.35 -> 1.0.36

NickCao committed:
Merge pull request #300497 from r-ryantm/auto-update/dovecot_fts_xapian

dovecot_fts_xapian: 1.7.9 -> 1.7.10

NickCao committed:
Merge pull request #300563 from DerDennisOP/bitwarden-directory-connector

bitwarden-directory-connector: 2024.3.1 -> 2024.3.2

NickCao committed:
Merge pull request #300539 from r-ryantm/auto-update/bash-supergenpass

bash-supergenpass: unstable-2020-02-03 -> unstable-2024-03-24

NickCao committed:
Merge pull request #300600 from Aleksanaa/go-musicfox

go-musicfox: 4.3.1 -> 4.3.3

NickCao committed:
Merge pull request #300598 from r-ryantm/auto-update/johnny-reborn-engine

johnny-reborn-engine: 0.30 -> 0.34

NickCao committed:
Merge pull request #300576 from r-ryantm/auto-update/werf

werf: 1.2.300 -> 1.2.301

NickCao committed:
Merge pull request #300638 from r-ryantm/auto-update/kopia

kopia: 0.16.0 -> 0.16.1

NickCao committed:
Merge pull request #300628 from r-ryantm/auto-update/sickgear

sickgear: 3.30.15 -> 3.30.16

NickCao committed:
Merge pull request #300623 from r-ryantm/auto-update/kubedb-cli

kubedb-cli: 0.42.1 -> 0.44.0

NickCao committed:
Merge pull request #300614 from r-ryantm/auto-update/vulkan-caps-viewer

vulkan-caps-viewer: 3.34 -> 3.40

NickCao committed:
Merge pull request #300613 from r-ryantm/auto-update/python312Packages.pytest-mypy-plugins

python312Packages.pytest-mypy-plugins: 3.1.1 -> 3.1.2

NickCao committed:
Merge pull request #300611 from r-ryantm/auto-update/conftest

conftest: 0.50.0 -> 0.51.0

NickCao committed:
Merge pull request #300609 from r-ryantm/auto-update/python312Packages.imbalanced-learn

python312Packages.imbalanced-learn: 0.12.0 -> 0.12.2

wolfgangwalther committed:
perl: fix spelling of filename

Ericson2314 committed:
Merge pull request #299347 from obsidiansystems/freebsd-by-name

FreeBSD: Use separate files for packages

superherointj committed:
Merge pull request #298938 from Ramblurr/init-davis

davis: init at 4.4.1

drupol committed:
Merge pull request #300661 from Raroh73/update/vscode/continue

vscode-extensions.continue.continue: 0.8.12 -> 0.8.22

drupol committed:
Merge pull request #298087 from andersrh/upgrade-github-desktop

github-desktop 3.3.6 -> 3.3.12

marsam committed:
Merge pull request #300218 from marsam/update-jekyll

jekyll: update dependencies

stigtsp committed:
Merge pull request #300687 from wolfgangwalther/perl-intepreter

perl: fix spelling of filename

marsam committed:
Merge pull request #300675 from r-ryantm/auto-update/act

act: 0.2.60 -> 0.2.61

marsam committed:
Merge pull request #300632 from r-ryantm/auto-update/oxlint

oxlint: 0.2.14 -> 0.2.15

marsam committed:
Merge pull request #300616 from r-ryantm/auto-update/chezmoi

chezmoi: 2.47.2 -> 2.47.3

marsam committed:
Merge pull request #299341 from marsam/update-tokio-console

tokio-console: 0.1.9 -> 0.1.10

marsam committed:
Merge pull request #300412 from r-ryantm/auto-update/golangci-lint

golangci-lint: 1.57.1 -> 1.57.2

NickCao committed:
Merge pull request #300607 from r-ryantm/auto-update/python312Packages.plexapi

python312Packages.plexapi: 4.15.10 -> 4.15.11

NickCao committed:
Merge pull request #300604 from r-ryantm/auto-update/quarkus

quarkus: 3.8.3 -> 3.9.1

NickCao committed:
Merge pull request #300589 from r-ryantm/auto-update/websocat

websocat: 1.12.0 -> 1.13.0

Commit:
Merge pull request #295594 from ShamrockLee/checkpoint-build-dontfixup

checkpointBuildTools.prepareCheckpointBuild: stop at `installPhase`

NickCao committed:
Merge pull request #300066 from NickCao/v2ray-domain-list-community

v2ray-domain-list-community: 20240316051411 -> 20240324094850

marsam committed:
Merge pull request #300244 from marsam/update-gollum

gollum: update dependencies

SomeoneSerge committed:
Merge pull request #300668 from GaetanLepage/testcontainers

python311Packages.testcontainers: 4.2.0 -> 4.3.0

philiptaron committed:
openafs: avoid top-level with

dotlambda committed:
Merge pull request #298777 from dotlambda/ytmdesktop

ytmdesktop: remove

flokli committed:
Merge pull request #299652 from philiptaron/issue-208242/nixos/lib/systemd

Avoid top-level `with ...;` in `nixos/lib/systemd*`

flokli committed:
Merge pull request #299717 from ElvishJerricco/systemd-debug-shell

nixos/systemd: Enable debug-shell.service.

jrhawley committed:
Merge branch 'add-vscode-extensions.snakemake.snakemake-lang' of https://github.com/jrhawley/nixpkgs into add-vscode-extensions.snakemake.snakemake-lang

drupol commented:
It looks like you accidentally mass-pinged a bunch of people, which are now subscribed
and getting notifications for everything in this pull request. Unfortunately, they
cannot be automatically unsubscribed from the issue (removing review request does not
unsubscribe), therefore development cannot continue in this pull request anymore.

Please create a new pull request, link back to this one and ping the
people actually involved in here over there. For the next time, remember to set your PR to draft status before rebasing. In draft status, you can preview the list of maintainers that are about to be requested for review, which allows you to sidestep this issue.

In order to avoid this in the future, there are instructions for how to properly
rebase between branches in our [contribution guidelines](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#rebasing-between-branches-ie-from-master-to-staging).
Setting your pull request to draft prior to rebasing is strongly recommended.
In draft status, you can preview the list of people that are about to be requested
for review, which allows you to sidestep this issue.
This is not a bulletproof method, though, as OfBorg still does review requests even on draft PRs.
